m3u8/model.py

```diff
@@ -637,10 +803,16 @@ def dumps(self):
 
         return ('#EXT-X-MEDIA:' + ','.join(media_out))
 
-    def __str__(self):
-        return self.dumps()
+    @classmethod
+    def _validate_boolean_attribute(cls, value, name):
```

Comment:
Since this is used just internally by the `Media` class, I think we should ditch the `@classmethod` decorator and make it an instance method. I would use a `@classmethod` decorator just to provide different methods of instantiating a class.